src/views/App.js

```diff
@@ -3,7 +3,6 @@ import React, { Fragment, Component } from 'react'
 import { decorate, observable, runInAction } from 'mobx'
 import { observer, inject } from 'mobx-react'
 import { Link } from 'react-router-dom'
-import DevTools from 'mobx-react-devtools'
```

Comment:
would you mind expanding on these changes a little? i'm just not sure what inspired them or what they are trying to achieve.

Reply:
So this just enables a small toolbar at the top right corner of the browser for mobx dev tools. It's just to help debug what mobx is updating and what not. The thing is, since it's on the top right of the browser, it also blocks the button to download which causes the tests to fail. 

![image](https://user-images.githubusercontent.com/23747649/66168399-dc308080-e602-11e9-9517-e7d06d7493d8.png)

Reply:
interesting. i don't think the tests have the `development` value set to `NODE_ENV` when they are running in Circle, do they? do the tests fail on Circle because of this issue, or is this just a problem while running locally?

Reply:
We actually don't have the tests running on Circle, so it's only failing locally

Reply:
I can try to run it again with the dev-tools and make sure that it's actually failing and it wasn't just something else

Reply:
Oh it looks like mobx react devtools is no longer supported after `mobx-react@6` because the standard react devtools can now do everything that mobx react devtools did. 

So even though we don't have that specific version of mobx-react, I think it's safe to remove since developers can now just use React devtools.